packages/text-annotator/src/SelectionHandler.ts

```diff
@@ -166,6 +158,8 @@ export const SelectionHandler = (
    * to the initial pointerdown event and remember the button
    */
   const onPointerDown = (evt: PointerEvent) => {
+    if (isContextMenuOpen) return;
```

Comment:
Hello, @rsimon 👋🏻 

Could you please clarify why the context menu guard was added for the `pointer` event handlers? With it in place, it takes 2 taps to dismiss the annotation selection. The first tap is needed to dismiss the context menu and the second one is to dismiss the annotation itself.

However, the annotation popup gets dismissed on the first tap too due to the `outside-press` reason! 
![image](https://github.com/user-attachments/assets/08520622-5297-4b3f-b4a3-80410be9c160)

That makes the annotation selection look like it's lagging behind 🤔 

https://github.com/user-attachments/assets/d9e3526c-dedb-4582-9a3c-a055968e79a3

Reply:
I see that it was added under the `Minor fix` commit https://github.com/recogito/text-annotator-js/commit/fb3d70bedba11660d64b479460094e951e530d38. However, it doesn't carry enough context 😅

Reply:
Aha... The "bug" manifested itself only when I ran the React example w/o the `isContextMenu` check: 
![image](https://github.com/user-attachments/assets/a8354cc5-ec5d-4e9c-86cc-77ed6f484dbc)

Reply:
Oh wait... The same exception exists even on the `main` branch when I run the React example on Android!